.github/workflows/ci.bazelrc

```diff
@@ -1,8 +1,14 @@
+common --build_metadata=ROLE=CI
+
 build --show_timestamps
 build --announce_rc
 build --color=yes
 build --curses=no
 build --terminal_columns=120
 build --verbose_failures
 
+build:remote-cache --remote_upload_local_results=true
+build:remote-cache --remote_instance_name=ci-runner-sel-instance-0
```

Comment:
@mickvangelderen you can choose whatever name for the instance that you want.

I use `oliver-macos-sel-local`. If you use the same remote instance name as CI, you can share remote cache -- assuming the remote env is close enough.

Reply:
You should be able to find the buildbuddy-api-key here:
https://app.buildbuddy.io/settings/org/api-keys